tests/e2e/page-objects/Signup.js

```diff
@@ -0,0 +1,16 @@
+import { Selector } from 'testcafe';
+
+export default class SignUp {
+    constructor() {
+        this.headerSignUp = Selector('a').withText('Sign up');
+        this.headerSignUp = Selector('a').withText('Log in');
```

Comment:
line 6 overrides line 5, so probably forgot to change name

Reply:
fixed